jose/backends/cryptography_backend.py

```diff
@@ -36,7 +36,8 @@ def __init__(self, key, algorithm, cryptography_backend=default_backend):
         self.hash_alg = {
             ALGORITHMS.ES256: self.SHA256,
             ALGORITHMS.ES384: self.SHA384,
-            ALGORITHMS.ES512: self.SHA512
+            ALGORITHMS.ES512: self.SHA512,
+            ALGORITHMS.ES256K: self.SHA256
```

Comment:
Nitpick: I'm a huge fan of trailing commas in lists to minimize the size of diffs (and also makes git blames a lot more easier). Please add a trailing comma to the `ALGORITHMS.E256K` line.